replicated-security/stride-ics-testnet-two-1/readme.md

```diff
@@ -0,0 +1,113 @@
+Follow along with Stride's block explorer here: https://ics-explorer.stride.zone/stride 
+
+For step 1, you can try using Stride’s joining script here: https://github.com/Stride-Labs/mainnet/blob/ics-testnet/ics-testnet/join_ics_testnet.sh 
+
+Otherwise you may manually join stride-ics-testnet-two-1 using these notes:
+* Joining instructions: https://github.com/Stride-Labs/mainnet/tree/ics-testnet/ics-testnet
+* Genesis file: https://raw.githubusercontent.com/Stride-Labs/mainnet/ics-testnet/ics-testnet/genesis.json
+* Pre-transition stride binary commit: `a3eff2dc`
+* Stride’s GitHub repository: https://github.com/stride-Labs/stride
+* Building instructions for stride’s binary: `make install`
+* Go version: 1.19
+* Persistent Peers = `"d747545dbab1eb86caff7ec64fca3b7f2ace07fd@stride-direct.testnet-2.stridenet.co:26656"`
+
+* Chain ID: stride-ics-testnet-two-1
+* Post-upgrade stride binary commit (run with this binary after the upgrade): [`5a45a4cfaef54aa0e875eb4e618bfae882d278a2`](https://github.com/Stride-Labs/stride/commit/5a45a4cfaef54aa0e875eb4e618bfae882d278a2)
+  * You can also build with [this Docker image](https://hub.docker.com/layers/stridelabs/ics-testnet/stride/images/sha256-3268198b39fa9e3b6107f352f49d28c5c78939e1147370b166f848dbd112186e?context=repo)
+ 
+<details><summary>Detailed steps for joining Stride Testnet</summary>
+<br>
+ 
+ _Courtesy of Stakecito_
+
+```sh
+git clone https://github.com/Stride-Labs/stride.git
+cd stride
+git checkout a3eff2dc
+make install
+strided init stride-node --chain-id stride-ics-testnet-two-1
+
+# Grab the genesis file
+curl -L https://raw.githubusercontent.com/Stride-Labs/mainnet/ics-testnet/ics-testnet/genesis.json -o $HOME/.stride/config/genesis.json
+```
+
+* Start stride node, node should start catching up
+* Node will panic at UPGRADE_HEIGHT_TBD
+* Stop the node
+
+```sh
+cd $HOME/stride
+
+git checkout 5a45a4cfaef54aa0e875eb4e618bfae882d278a2
+
+make install
+```
+
+replace the binary
+
+```sh
+mkdir -p $HOME/.sovereign/config
+
+curl -L https://github.com/cosmos/testnets/raw/master/replicated-security/stride-ics-testnet-two-1/genesis.json -o $HOME/.sovereign/config/genesis.json
+```
+</details>
+
+<details><summary>Detailed steps for transitioning Stride node from non-validator on Stride testnet to validator on consumer chain</summary>
+<br>
+
+_Thanks to Bosco from Silk Nodes_
+
+Download v11 Binary
+```sh
+cd stride
+git pull
+git checkout 5a45a4cfaef54aa0e875eb4e618bfae882d278a2
+make install
+
+#Should be v11
+strided version
+```
+
+Make directories in cosmovisor and copy binaries
+```
+mkdir -p $HOME/.stride/cosmovisor/upgrades/v11/bin/
+cp $HOME/go/bin/strided $HOME/.stride/cosmovisor/upgrades/v11/bin/
+```
+
+Download new Sovereign genesis
+```
+mkdir -p $HOME/.sovereign/config/
+wget -O $HOME/.sovereign/config/genesis.json https://cdn.discordapp.com/attachments/1064857924402413600/1116022613966336041/genesis_new.json
```

Comment:
Can you confirm the path will still be `$HOME/.sovereign/config/` ?
Also, we should not list a URL until the new genesis is available- putting this up ahead of the spawn time will probably lead to people downloading the first rehearsal's patched genesis file.